cypress/cypress-integration/support/commands.ts

```diff
@@ -24,6 +24,10 @@ Cypress.Commands.add('showsSuccessMessage', (shows = true) => {
   }
 });
 
+Cypress.Commands.add('getByTestId', (testId: string) => {
```

Comment:
What's the benefit of adding this command?

Reply:
Thanks for the link! I don't love obfuscating such a simple command to be honest, but I'm willing to try it out.

Reply:
I am with Rachel as well and do not love the obfuscating of a simple command. I generally don't like adding commands onto the global cypress object and prefer generic javascript functions. 

Open to changing my mind if you feel strongly about it though Cody.

Reply:
I'm with Cody on this one, this is what Cypress Commands are for